### DISCUSSION TITLE
How can I specify a file extension and the compression level / quality for my saved screenshots?

### DISCUSSION THREAD

**Discussion by Yasand123** [flameshot-org/flameshot#3538] [Q&A] [answered]
Tl;dr: I want .jxl with 80 compression level/quality for saved screenshots.

Detailed:
I want my saved fullscreen screenshots to be saved as a specific file type. In my case I want it to be .jxl (JpegXL) and I want to specify the quality argument as well. In my case it would be `-q 80`. Is this possible within flameshot? I read the documentation and I couldn't find anything like this.

Note: I take screenshots with command arguments assigned to keyboard keys, not through the GUI.

**Comment by SamtheEevee69** [comment]
I searched google for this
Look's like we're all alone
;w;

**Comment by mmahmoudian** [comment, marked as answer]
Sorry for the late reply. I somehow have missed this discussion.

Flameshot doesn't support image compression like that for now. I suggest getting the raw image from Flameshot (e.g `flameshot gui --raw`) and pipe it into imagemagic.

**Reply by SamtheEevee69** [reply]
Such an easy thing
Why is it not supported from the main app?
We need extra third-party apps for one app to work 100% of the power
Whyyyy

**Reply by Yasand123** [reply]
Perfect. Thank you. Here's the command I come up with in case someone wants it:
`flameshot gui --raw | magick convert - -quality 85 ~/Pictures/Screenshots/"$(date '+%Y-%m-%d_%H-%M').jxl"`

It should follow the same naming scheme as flameshot (at default settings). Make sure to change the output location.

**Reply by Yasand123** [reply]
@mmahmoudian 
One more question if you don't mind; is there a simple way to get a similar notification using this method?

`flameshot full --raw | magick convert - -quality 85 ~/Pictures/Screenshots/"$(date '+%Y-%m-%d_%H-%M').jxl"` this while does what I want will not make a notification popup like if you used flameshot directly.
